.github/README.md

```diff
@@ -0,0 +1,109 @@
+# Roadkit
+
+Swift Package for simple and fast integration of Roadkit into iOS and macOS apps.
+
+
+---
+
+
+## Content
+- [Features](#features)
+- [Installation](#installation)
+- [How to Use](#how-to-use)
+
+
+## Features
+- [x] Fetch open topics for your app
+- [x] Post a new topic (Feature, Improvement or Bug)
+- [x] Vote for a topic
+- [x] Undo votes for a topic
+- [x] Get if the current user has voted for a topic
+
+
+## Installation
+#### Requirements
+The SDK requires an account for AppState.dev with a registered app.
+
+- iOS 14.0+ / macOS 12.0
+- Xcode 13+
+- Swift 5+
+
+#### Swift Package Manager
+In Xcode, go to `File > Add Packages` and add `https://github.com/hyperlinkgroup/roadkit-sdk-ios`. Add the package to your desired targets.
+
+
+## How to Use
+#### Setup
+To setup Roadkit, you need to initiate the SDK with your project's ID and a UserID. The ProjectID can be found in your macOS app, in the info tab of the app. The UserID is the ID of your current user. To setup Roadkit you need to `import Roadkit` on top of your file and then call:
+```Swift
+RoadkitManager.shared.setupRoadkit(projectID: <String>, userID: <String>)
+```
+
+If your user changes, you can call the following function to update the UserID:
+```Swift
+RoadkitManager.shared.updateUserID(with: <String>)
+```
+
+**Important**: The UserID is optional and if you don't provide one, it will be sent as an empty string. We strongly recommend sending a UserID (if you don't authenticate your users, you can create a UUID and store it in your UserDefaults or Keychain). Keep in mind that if no ID is provided, the check wether your user has voted for a specific topic will always return false and voting does not work.
```

Comment:
"and voting does not work"
- wo wird das denn abgefangen? hier im Package? Über die Route kann ich nämlich mit leerem String voten, dann wird `:userId` ins Array hinzugefügt. Ich denk mal das ist ein Bug, legst du dafür ein Ticket an?

Reply:
Aaah okay das ist dann ein Bug in den Functions. Ich hatte hier nur daran gedacht, dass wir dann keine UserID haben, die abgeglichen werden kann. ROK-50 angelegt